packages/notifier/src/publish-kit.js

```diff
@@ -4,10 +4,19 @@
 import { HandledPromise, E } from '@endo/eventual-send';
 import { makePromiseKit } from '@endo/promise-kit';
 import { Far } from '@endo/marshal';
+import { M } from '@agoric/store';
+import { canBeDurable, vivifyFarClassKit } from '@agoric/vat-data';
 
 import './types.js';
 
+const { details: X, quote: q } = assert;
+
 const sink = () => {};
+const makeQuietRejection = reason => {
+  const rejection = HandledPromise.reject(reason);
+  rejection.catch(sink);
```

Comment:
In this particular case, the `.catch` is safe. But best to avoid bad habits and reinforce good ones.

```suggestion
  E.when(rejection, undefined, sink);
```

Alternatively, is `makeQuietRejection` a sufficiently general purpose and reusable abstraction that it should move to one of: eventual-send, promise-kit, or internal ?

Reply:
Updated. We should keep the possibility in our back pocket, but I don't think `makeQuietRejection` is needed outside of this file for now.